config.json

```diff
@@ -431,6 +445,20 @@
           "text_formatting"
         ]
       },
+      {
+        "slug": "rotational-cipher",
+        "name": "Rotational Cipher",
+        "uuid": "3adbc85c-7e7e-4122-a37d-c843196bc8af",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4,
```

Comment:
I bumped this difficulty up since it uses `malloc`. This keeps these exercises away from the beginning of the track.